packages/github/README.md

```diff
@@ -55,3 +55,19 @@ const newIssue = await octokit.issues.create({
   body: 'Hello Universe!'
 });
 ```
+
+## Webhook Payload Typescript Definitions
+`@octokit/webhooks` provides complete typescript definitions for the payloads associated with each event. You can cast the provided payload to these types for more type information as needed.
+
+To do this, first install the npm module `npm install --save @octokit/webhooks`
+
+Then, assert the type based on the eventName
```

Comment:
> assert the type

consider "cast to the type". Assert implies validation - connotation from common unit test terminology and also assert module built-in to nodejs

Reply:
It isn't a cast though:
https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions